DEVELOPER.md

```diff
@@ -30,9 +30,10 @@ Below is a list of additional documentation to aid the development process:
 
 - [Performance testing Envoy with `tcmalloc`/`pprof`](https://github.com/envoyproxy/envoy/blob/master/bazel/PPROF.md)
 
+- [Porting Envoy and common porting issues](https://github.com/envoyproxy/envoy/blob/master/bazel/PORTING.md)
```

Comment:
nit: remove `bazel/` part of path since `PORTING.md` is in the root of the repo

Reply:
This is actually a reminder-to-self :)  I've been torn since starting whether PORTING.md lives in root or under bazel/ --- @htuch or @mattklein123 do you have an opinion one way or another?

Reply:
Top-level is fine, Bazel really should have some relationship with Bazel, e.g. README.md or the external deps doc.

Reply:
@wrowe fix the link?